src/panels/lovelace/views/hui-view-header.ts

```diff
@@ -38,6 +39,14 @@ export class HuiViewHeader extends LitElement {
 
   @property({ attribute: false }) public viewIndex!: number;
 
+  @query(".badges.scroll") private _badgesScrollContainer!: HTMLElement;
+
+  @state() private _badgesScrollMouseDown = false;
+
+  @state() private _badgesScrollStartX = 0;
+
+  @state() private _badgesScrollLeft = 0;
```

Comment:
These don't need to trigger a render cycle when changed, so you should drop the `@state()` decorator.